composer.json

```
@@ -127,14 +127,23 @@
             "drush config:set --yes system.logging error_level verbose",
             "drush pm:enable --yes default_content"
         ],
+        "drupal:quick-start": [
+            "Composer\\Config::disableProcessTimeout",
+            "@php web/core/scripts/drupal quick-start"
```

I think we might need to keep the `-d max_execution_time=0` here.

```suggestion
            "@php -d max_execution_time=0 web/core/scripts/drupal quick-start"
```

---

I don't think we do, actually. I tested it last week, and it will run for hours without it, even without any requests coming in.

---

Welp, if you tested it, then I believe you!

---

You know, I just realized something -- the script description is inaccurate. As written here, `drupal:quick-start` will NOT install Drupal CMS; it will install Standard, I think.

So it actually does need to invoke `@composer drupal:install`. Then `post-create-project-cmd` can just be an alias, essentially, for `drupal:quick-start`.

---

If we do that (invoke `drupal:install` at the beginning of `drupal:quick-start`), it will destroy the database and reinstall the site from scratch every time (tested and confirmed), defeating the whole purpose. What if we just update the description to be more accurate?

```diff
-        "drupal:quick-start": "Installs Drupal CMS (if not already installed) and runs the PHP webserver."
+        "drupal:quick-start": "Runs Drupal CMS using the PHP webserver and opens it in the default browser."
```

---

Well...I dunno about that. To me, a "quick start" _suggests_ that it starts you off quickly "from the beginning", as it were.

How about we rename the command to `drupal:serve` instead, for clarity?

---

Yeah, I like that. Why don't I keep following Drush's naming conventions and call it `drupal:run-server` with aliases of `rs` and `serve`? It's ready for you again.